### DISCUSSION TITLE
When I create an app and use the MultiSelect component with the data source set to Connection or Eval, the page crashes when I try to run it.

### DISCUSSION THREAD

**Discussion by YADROOKIE** [windmill-labs/windmill#1692] [Questions]
1、create app  select MultiSelect component
2、Item datasource use  connect or eval 
![image](https://github.com/windmill-labs/windmill/assets/43526965/55326212-3cdd-477c-b80d-d625c730efc5)
3、run app  then page crashes 
![image](https://github.com/windmill-labs/windmill/assets/43526965/6eda8779-35fa-4ec0-9a28-074a96fb1ee2)

**Comment by rubenfiszel** [comment]
I cannot easily reproduce, what version of windmill are you on ? Does it happen everytime by following those exact instructions ?

**Reply by YADROOKIE** [reply]
yeah,  when i do it in  windmill-cloud  It can be replicated easily

**Reply by rubenfiszel** [reply]
Got it, I just did a fix, when it deploys let me know if you still experience this

**Reply by YADROOKIE** [reply]
thank you
